es-app/src/guis/GuiSettings.cpp

```diff
@@ -45,10 +45,18 @@ bool GuiSettings::input(InputConfig* config, Input input)
 			delete window->peekGui();
 		return true;
 	}
-	
+
 	return GuiComponent::input(config, input);
 }
 
+HelpStyle GuiSettings::getHelpStyle()
+{
+	HelpStyle style = HelpStyle();
+	// TODO : Find a better way to get theme
```

Comment:
I'd rather not have more TODOs in the code - if you think something needs addressing perhaps it's worth doing it now?